app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -108,3 +108,84 @@ legend p {
     width: 100%;
   }
 }
+
+.select2-choice {
+  abbr {
+    background: url("select2.png") right top no-repeat;
+  }
+
+  .select2-arrow {
+    b {
+      background: url("select2.png") no-repeat 0 1px
+    }
+  }
+}
+
+.navbar-brand {
+  background: url("govuk_admin_template/header-crown.png") 0 .67em no-repeat;
+}
+
+// stylelint-disable function-linear-gradient-no-nonstandard-direction 
```

Comment:
Seems like, from [this warning in the stylelint docs](https://stylelint.io/user-guide/ignore-code/#:~:text=WARNING,or%20value%20lists.), that maybe it would be best to use `/* */` comments?

(also applies later in this file)

Reply:
> Seems like, from [this warning in the stylelint docs](https://stylelint.io/user-guide/ignore-code/#:~:text=WARNING,or%20value%20lists.), that maybe it would be best to use `/* */` comments?
> 
> (also applies later in this file)

Yes I agree except `/* */` fails the linter itself with the message: "Expected // for comments instead of /*  scss/comment-no-loud", which seems to contradict its own documentation. To get round this I've added a comment to the one already in place at the top to disable that one.